sig-architecture/annual-report-2021.md

```diff
@@ -0,0 +1,102 @@
+# 2021 SIG Architecture Annual Report
+
+## Operational
+
+**How are you doing with operational tasks in sig-governance.md?**
+
+**Is your README accurate? have a CONTRIBUTING.md file?**
+
+ - Yes, our README is accurate. We do not have a CONTRIBUTING.md file, and so will add one.
+
+**All subprojects correctly mapped and listed in sigs.yaml?**
+
+ - Yes.
+
+**What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?**
+
+ - Our bi-weekly meeting is normally well attended, with approximately 20 individuals on a typical day, though it varies from a handful of folks to 40 or 50 depending on topics.
+
+ - Subproject meetings are smaller, generally in the half dozen range.
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?**
+
+ - We have a rotating schedule of subproject updates in the bi-weekly community meeting. Our OWNERS files are up-to-date as well.
+
+**How does the group get updates, reports, or feedback from working groups? Are there any springing up or being retired? Are OWNERS files up to date in these areas?**
+
+ - We have not been getting updates from the working groups on any regular cadence.
+ - We lost track of the WGs we sponsored. We will make it a priority to get updates in the next upcoming meetings.
```

Comment:
Can the known WGs be listed here?

Reply:
Looked up because I was curious myself:
- wg-api-expression
- wg-component-standard
- wg-k8s-infra
- wg-naming
- wg-policy
- wg-reliability